app/views/checklist_tests/_field_values.html.erb

```diff
@@ -3,21 +3,102 @@
 # local variables
 #
 #   field_values   (Hash)
+#   product_test (ChecklistTest)
 
 %>
 <% if negation %>
```

Comment:
Here it looks like you have similar html for the modal 3 times, which also looks like the same html for the modal in `_checklist_measure.html.erb` so that can probably be refactored. The best way to do it would be to put the modal in a partial as mentioned in another comment.